packages/nimble-components/src/table/models/tests/table-validator.spec.ts

```diff
@@ -238,6 +242,94 @@ describe('TableValidator', () => {
         });
     });
 
+    describe('column config validation', () => {
+        const columnConfigurations: {
+            columns: TableColumnValidationTest[],
+            isValid: boolean,
+            name: string
+        }[] = [
+            {
+                columns: [
+                    Object.assign(
+                        document.createElement(tableColumnValidationTestTag),
+                        { foo: true, bar: true }
+                    ) as TableColumnValidationTest,
```

Comment:
expect we can remove the `as TableColumnValidationTest` casts in the file after adding it to the element map

Reply:
Yes